### ISSUE TITLE
Use GraphQL for tailwind demo

### ISSUE THREAD

**Issue by jeffsee55** [tinacms/tina-graphql-gateway#199]
Right now the tina.io homepage links to the inline editing version of this from the hero. @spbyrne has plans to replace that and update the gif to reflect sidebar editing focus, and the work for using sidebar seems pretty close to done in this repo https://github.com/tinacms/tina-tailwind-sidebar-demo. 

We've got a few things moving around in regard to what's needed for this ticket but at a high-level we want to do these things:

- [ ] Move the code into this repo, for easier maintenance - Not critical but at the same time not a compelling reason _not_ to right?
- [x] Swap out the use of manual forms for GraphQL. This could be a good exercise for someone less familiar with GraphQL to pair on.
- [x] Some of the "content" is currently being pulled in via `import theme from "../mytheme.json"` - this would need a dev server to be running for it to work on a hosted server. So the assumption is we'll want to change that to be driven by GraphQL fetch or at least `fs.readFile`. 

It also sort of feels natural to continue to use json for the theme file, though not strictly necessary. We might as well wait for that to be supported

**Comment by logan-anderson** [comment]
All of the editing logic is not moved to Tina Cloud. However it currently does not work with Tina Cloud as we need Tina Cloud to update its gateway package to be on the latest version. (We need the update that adds JSON support)

PR: https://github.com/tinacms/tina-tailwind-sidebar-demo/pull/2
cc @jamespohalloran

**Comment by logan-anderson** [comment]
@jeffsee55 I do not have push access. I have a PR ready of I can gain access.

**Comment by jamespohalloran** [comment]
@logan-anderson Just added you onto that repo

**Comment by logan-anderson** [comment]
@jamespohalloran  thanks! The PR is done

https://github.com/tinacms/teams-serverless/pull/433

**Comment by logan-anderson** [comment]
This issues is currently block by https://github.com/tinacms/teams-serverless/issues/440

**Comment by jeffsee55** [comment]
This is also blocked by the fact that we can't persist this data on the server in an unauthenticated state. More discovery is needed about how we might best handle this.

**Comment by jamespohalloran** [comment]
> This issues is currently block by tinacms/teams-serverless#440

That's now been merged!

**Comment by jeffsee55** [comment]
We chatted about this work and decided that we need to support it in 2 ways for now. First - keep the external repo which @spbyrne built as-is. Meaning "open authoring"-like experience remains intact. Second - merge @logan-anderson's work in to the `examples` directory along with the starter. That ticket is tracked [here](https://github.com/tinacms/tina-graphql-gateway/issues/243). 

The downside here is that the 2 examples may diverge a bit. But in general we're not too worried about it, when someone wants to try the demo that's linked from the home page we just point them to the one in this repo instead of the one that's actually running. 

cc @mitchmac does that sound like an accurate representation of what we discussed?
